### ISSUE TITLE
Mobilenet v2 Quantized TOCO failed!

### ISSUE THREAD

**Issue by WenguoLi** [tensorflow/models#4137]
Mobilenet v2 Quantized TOCO failed, 

bazel-bin/tensorflow/contrib/lite/toco/toco '--input_file=/home/apuser/tmp/models/flowers/mobilenet_v2_1.0_224_quan/frozen_graph.pb' '--output_file=/home/apuser/tmp/models/flowers/mobilenet_v2_1.0_224_quan/tflite_model.tflite' '--input_format=TENSORFLOW_GRAPHDEF' '--output_format=TFLITE' '--inference_type=QUANTIZED_UINT8' '--input_shape=1,224,224,3' '--input_array=input' '--output_array=MobilenetV2/Predictions/Reshape_1' '--std_value=128' '--mean_value=127' '--dump_graphviz=/home/apuser/tmp/models/flowers/mobilenet_v2_1.0_224_quan'

INFO: Running command line: 
2018-05-01 10:53:26.254545: I tensorflow/contrib/lite/toco/graph_transformations/graph_transformations.cc:39] Before Removing unused ops: 1207 operators, 1752 arrays (0 quantized)
2018-05-01 10:53:26.300224: I tensorflow/contrib/lite/toco/graph_transformations/graph_transformations.cc:39] Before general graph transformations: 1207 operators, 1752 arrays (0 quantized)
2018-05-01 10:53:26.582437: I tensorflow/contrib/lite/toco/graph_transformations/graph_transformations.cc:39] After general graph transformations pass 1: 120 operators, 228 arrays (1 quantized)
2018-05-01 10:53:26.584527: I tensorflow/contrib/lite/toco/graph_transformations/graph_transformations.cc:39] Before pre-quantization graph transformations: 120 operators, 228 arrays (1 quantized)
2018-05-01 10:53:26.585399: I tensorflow/contrib/lite/toco/graph_transformations/graph_transformations.cc:39] After pre-quantization graph transformations pass 1: 67 operators, 175 arrays (1 quantized)
2018-05-01 10:53:26.586243: F tensorflow/contrib/lite/toco/tooling_util.cc:1464] 
**Array MobilenetV2/expanded_conv_2/add, which is an input to the Conv operator producing the output array MobilenetV2/expanded_conv_3/expand/Relu6, is lacking min/max data, which is necessary for quantization**. Either target a non-quantized output format, or change the input graph to contain min/max information, or pass --default_ranges_min= and --default_ranges_max= if you do not care about the accuracy of results.

**Comment by WenguoLi** [comment]
I find there are min/max data in graph:
![screenshot from 2018-05-01 16-52-01](https://user-images.githubusercontent.com/31765154/39467406-8ed5e460-4d60-11e8-86d5-fd6f6b6b1874.png)

I use the [export_inference_graph.py](https://github.com/tensorflow/models/blob/master/research/slim/export_inference_graph.py) to export graph, and add  the following code
```
    if FLAGS.quantize:
      tf.contrib.quantize.create_eval_graph()
```
to #121 lines.

**Comment by WenguoLi** [comment]
I had checked freeze_graph. I think it is ok!

bazel-bin/tensorflow/tools/graph_transforms/summarize_graph     --in_graph=${TRAIN_DIR}/frozen_graph.pbFound 1 possible inputs: (name=input, type=float(1), shape=[?,224,224,3]) 
No variables spotted.
Found 1 possible outputs: (name=MobilenetV2/Predictions/Reshape_1, op=Reshape) 
Found 2264689 (2.26M) const parameters, 0 (0) variable parameters, and 0 control_edges
Op types used: 562 Identity, 544 Const, 156 Mul, 114 Add, 106 **FakeQuantWithMinMaxVars**, 52 Sub, 52 Rsqrt, 36 Conv2D, 35 Relu6, 19 Reshape, 17 DepthwiseConv2dNative, 1 Placeholder, 1 BiasAdd, 1 Shape, 1 Softmax, 1 Squeeze, 1 AvgPool
To use with tensorflow/tools/benchmark:benchmark_model try these arguments:
bazel run tensorflow/tools/benchmark:benchmark_model -- --graph=/home/apuser/tmp/models/flowers/mobilenet_v2_1.0_224_quan/frozen_graph.pb --show_flops --input_layer=input --input_layer_type=float --input_layer_shape=-1,224,224,3 --output_layer=MobilenetV2/Predictions/Reshape_1

**Comment by OdingdongO** [comment]
i have the same problem with you,do you fix it now? @WenguoLi

**Comment by WenguoLi** [comment]
@OdingdongO ,
Please see [#19014](https://github.com/tensorflow/tensorflow/issues/19014)
